### ISSUE TITLE
NVDA does not read result in Windows Calculator

### ISSUE THREAD

**Issue by president731** [nvaccess/nvda#15387]
### Steps to reproduce:
1. Bring up Microsoft's built-in Windows Calculator.
2. Type  the numbers and appropriate operators to add, subtract, whatever.
3. Hit enter to get the result.

### Actual behavior:

Multiplying 6*4 often will either read 4 or nothing at all when you hit enter to get the result.

### Expected behavior:

When you hit the enter key to get the result, NVDA should say the result.

#### NVDA installed/portable/running from source:
installed on hard drive in installer default path

#### NVDA version:
NVDA 2023.3.2 2023.3.2.0.29051

#### Windows version:
Windows 7 Home Premium Service Pack 1 64-bit

#### Name and version of other software in use when reproducing the issue:
Windows 7's built-in Calculator

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
The prior version of NVDA 2023.3.1 did not do this.

#### If NVDA add-ons are disabled, is your problem still occurring?
yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by Adriani90** [comment]
Closing as duplicate of #15230.
